package.json

```diff
@@ -7,8 +7,8 @@
     "build": "remix vite:build",
     "dev": "remix vite:dev",
     "start": "remix-serve ./build/server/index.js",
-    "lint": "biome check --apply .app/",
-    "lint:ci": "biome ci",
+    "lint": "biome check . --config-path ./",
```

Comment:
we don't need to specify the config path, biome automatically finds `biome.json`. 

`biome check --apply .app/` tells the cli to use biome to check against lint and formatting rules and apply fixes for files in the app folder. (I think its unnecessary to check more than the app folder)

Is there any reason you'd prefer to check the entire codebase?

Reply:
> we don't really need to specify the config path, biome automatically finds `biome.json`. Were you running into an issue of it not recognizing the lint rules?
> 
> `biome check --apply .app/` tells the cli to use biome to check against lint and formatting rules and apply fixes for files in the app folder. (I think its unnecessary to check more than the app folder)
> 
> Is there any reason you'd prefer to check the entire codebase?

right, it automatically resolve the config file path, I somehow confused it